indico/modules/events/registration/templates/display/participant_list.html

```diff
@@ -11,6 +11,14 @@
     {% endif %}
 {% endblock %}
 
+{% block description %}
+    {%- if tables|length > 1 -%}
+        {% trans -%}
+            The lists of participants grouped by the registration form they used to registered to the event.
```

Comment:
"used to registered"? Also, I don't think this description is very useful - it is way too technical. Do we need a description there at all?